urlscan/plugin.spec.yaml

```diff
@@ -332,6 +343,11 @@ actions:
         type: integer
         description: Total number of results returned
         required: false
+      has_more:
```

Comment:
The pagination here seems like it could be tricky to use - is there any way we can make this easier for the user?

Reply:
@mrinehart-r7 Yes, it is possible to make it easier for the user, but it needs to rewrite whole pagination in this action. We can create a new ticket for this if you want :) We add here 'has_more' for better inform user that there is more results to returned. When we test this action we noticed that this part is missing and we add this, because it could be usefull for user.